stdlib/LinearAlgebra/src/symmetric.jl

```diff
@@ -254,11 +254,19 @@ end
     end
 end
 
+Base._reverse(A::Symmetric, dims::Integer) = (reverse((Matrix(A)); dims))
+Base._reverse(A::Symmetric, dims::Colon) = Symmetric(reverse(Matrix(A)))
+Base._reverse!(A::Symmetric, dims::Colon) = (reverse!(A.data); A)
```

Comment:
This implementation requires swapping uplo which is not possible because Symmetric is an immutable struct. You could delete this and let it keep erroring for now, or perform a more sophisticated reverse operation that reverses diagonal-wise.